server/aws-lsp-codewhisperer/src/language-server/inline-completion/codeWhispererServer.ts

```diff
@@ -143,16 +102,6 @@ const mergeSuggestionsWithRightContext = (
     })
 }
 
-interface AcceptedInlineSuggestionEntry extends AcceptedSuggestionEntry {
```

Comment:
move to https://github.com/aws/language-servers/pull/2028/files#diff-ad1f00e8de4b420f86052a8e2e8252a00aaaf2979c995a69a26c99b14ea585f7R16

Reply:
data class should live in such initialization file (codewhispererServer.ts)